fetch-content.sh

```diff
@@ -0,0 +1,42 @@
+#!/bin/bash
+
+# Die on any error for Travis CI to automatically retry:
+set -e
+
+function download_url() {
+  URL="$1"
+  LOCAL_FILE="${1##*/}"
+  if [ -x "$(type -P wget)" ]; then
+    wget -O "${LOCAL_FILE}" "${URL}" --no-check-certificate
+  elif [ -x "$(type -P curl)" ]; then
+    curl -f -o "${LOCAL_FILE}" "${URL}"
+  else
+    echo "No supported download method found." 1>&2
+    return 1
+  fi
+}
+
+if [ "$(uname)" == "Darwin" ]; then
+  DIR="$HOME/Library/Application Support/OpenRA/Content/ra2"
+else
+  DIR="$HOME/.config/openra/Content/ra2/"
+fi
+
+#if the directory already exists then exit
+[ ! -d "${DIR}" ] && mkdir -p "${DIR}" || {
+  cd "${DIR}"
+  #only exit if there's existing content
+  if ls *.mix &> /dev/null; then
+    exit 0
+  fi
+}
+echo "Downloading mod content"
+cd "${DIR}"
+#download the file else exit non-zero
+download_url "https://xwis.net/dl/Red-Alert-2-Multiplayer.exe"
+7z e Red-Alert-2-Multiplayer.exe
+download_url "https://xwis.net/dl/Yuris-Revenge-Multiplayer.exe"
```

Comment:
This was removed last year from XWIS when Olaf got fed up with people abusing it for playing on CnCNet.

Reply:
Just drop this commit IMO.

Reply:
I see. Makes sense.